.ebextensions/00_ami.config

```diff
@@ -15,6 +15,33 @@ packages:
     GraphicsMagick: []
     wget: []
 
+# Additional dependencies not available through yum.
+files:
+  /tmp/ffmpeg-release-64bit-static.tar.xz:
+    mode: "000644"
+    owner: root
+    group: root
+    authentication: "S3Auth"
+    source: https://s3-us-west-2.amazonaws.com/elasticbeanstalk-us-west-2-234923831700/suitec-preview-service/dependencies/ffmpeg-release-64bit-static.tar.xz
+  /tmp/ghostscript-9.20-linux-x86_64.tgz:
```

Comment:
@paulkerschen this would be a good spot for CloudFront URL, if it were available. Is the following an option?
```
    source: https://mybucket.s3.amazonaws.com/.../tmp/ghostscript-9.20-linux-x86_64.tgz
```